rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeFromTryCatchTypeRector/Fixture/skip_doc_type.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeFromTryCatchTypeRector\Fixture;
+
+final class SimpleTryCatch
+{
+    public function run()
+    {
+        try {
+            return 1;
+        } catch (\Exception $e) {
+            return $this->getIntDoc();
+        }
+    }
+
+    /**
+     * @return int
+     */
+    private function getIntDoc()
+    {
+        return mt_rand(0, 1) ? 'string' : 1;
```

Comment:
This skipped because of Union, not because of docblock.

To actual single type and native, use getNativeType